pysatMissions/tests/test_instruments.py

```diff
@@ -33,6 +33,8 @@
 for inst in instruments['download']:
     if 'sgp4' in inst['inst_module'].name:
         instruments['sgp4'].append(inst)
+    if 'skyfield' in inst['inst_module'].name:
+        instruments['sgp4'].append(inst)
```

Comment:
This means that skyfield instruments are added to the list of sgp4 instruments right?

Reply:
Yes. This is for tests that are not run on ephem